content/docs/user-guide/scripting/lua/_index.md

```diff
@@ -6,18 +6,24 @@ toc: true
 weight: 100
 ---
 
-**Topics**
-+ [Adding Lua Scripts to Component Entities](/docs/user-guide/editor/add-lua-script)
-+ [Properties Table](properties/)
-+ [Using EBuses in Lua](ebus/)
-+ [Lua Editor](lua-editor/)
-+ [Debugging Lua Scripts](debugging-scripts/)
-+ [The Lua Environment (Advanced)](environment/)
-
-You can use Lua in O3DE to facilitate quick iteration of your game project. Lua is a powerful, fast, lightweight, embeddable scripting language. When you construct new gameplay and game systems, you can run your changes immediately, without compiling your source code.
+You can use Lua in **Open 3D Engine** (O3DE) to facilitate quick iteration of your game project. Lua is a powerful, fast, lightweight, embeddable scripting language. When you construct new gameplay and game systems, you can run your changes immediately, without compiling your source code.
```

Comment:
```suggestion
You can use Lua in **Open 3D Engine (O3DE)** to facilitate quick iteration of your game project. Lua is a powerful, fast, lightweight, embeddable scripting language. When you construct new gameplay and game systems, you can run your changes immediately, without compiling your source code.
```

Reply:
@JwGedit This is another case that I could add to the style guide; I may have received other guidance from editors.  A second example I found in the wild: **Open 3D Engine (O3DE) Asset Editor**.